docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -1815,6 +1815,59 @@ try {
 }
 ```
 
+### User Notification Settings [#user-notification-settings]
+
+<Banner title="Private beta">
+
+User notification settings is currently in private beta. If you would like
```

Comment:
It will be open beta, anyone can download the package and it will be activated for all